<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>devInfo</class>
 <widget class="QWidget" name="devInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>340</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>8</y>
     <width>308</width>
     <height>288</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="verticalSpacing">
     <number>1</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Serial Number:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="lb_serialNumber">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Not Defined</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_6">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Lidar Version:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="lb_versionLidar">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Not Defined</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_7">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Video Version:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="lb_versionVideo">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>video version</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_8">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Linux Version:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="lb_versionLinux">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>linux version</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_9">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>10</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>10</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_10">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Video Count:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="lb_countVideo">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>video count</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_11">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Image Count:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="lb_countImage">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>image count</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_12">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Ticket Count:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="lb_countTicket">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>ticket count</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_13">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>10</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>10</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_14">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Expiration:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLabel" name="lb_expiration">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>expiration</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_15">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Authority:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLabel" name="lb_authority">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>authority</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_16">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Certification:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLabel" name="lb_certification">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>certification</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
